packages/react/src/views/AttachmentHandler/Attachment.js

```diff
@@ -44,6 +51,42 @@ const Attachment = ({ attachment, host, type, variantStyles = {} }) => {
       />
     );
   }
+  if (attachment && attachment.attachments[0]?.image_url) {
```

Comment:
Why introduce redundant rendering of a component? Instead, you can add the required condition where we are already using `ImageAttachment`, `AudioAttachment`, and `VideoAttachment`.

Reply:
It was easier for me to implement like this and this is also a conditional based rendering